.vib/mysql/goss/goss.yaml

```diff
@@ -0,0 +1,34 @@
+command:
+  mysql-env:
+    exec: env
+    exit-status: 0
+    stdout:
+      - "MYSQL_DATABASE=my_database"
```

Comment:
Given we are testing `MYSQL_DATABASE` has an explicit value, I would add `auth.database=my_database` to the `runtime_parameters` of our VIB pipeline to avoid any issues in the future.

Reply:
I´ve added and customised this parameter. Thanks!